### DISCUSSION TITLE
problemi/mancanze varie

### DISCUSSION THREAD

**Discussion by tommasiangelo01** [iisgiua/giuaschool#606] [Richieste di aiuto]
Buonasera ,è da ben 3 anni che i genitori e gli alunni stessi si lamentano che non possono vedere la media di ciascuna materia ,così come non possono prendere visione del fuoriclasse (cosa importantissimo ,dal momento che se dovesse succedere qualsiasi cosa i genitori dovrebbero sapere che in quel momento il proprio figlio era fuoriclasse per motivi x,così come il fatto delle supplenze dove il docente che sostituisce firma senza inserire chi sta sostituendo e la materia di insegnamento di questo che risulta assente.Sono piccolezze che ormai sono presenti in tutti i registri elettronici del mondo ,per cui sarebbe bello se anche qui fossero integrati,così come la possibilità di stampare gli elenchi delle classi o elenchi docenti ,ata,etc...

**Comment by trinko** [comment]
Vero.
Sarebbe anche molto bello se qualcuno decidesse di contribuire, nello spirito dell'_open source_, pagando un programmatore per sviluppare le funzionalità che ritiene prioritarie...
